server/src/main/java/de/hellfish/ngrok/server/ClientList.java

```diff
@@ -0,0 +1,50 @@
+package de.hellfish.ngrok.server;
+
+import lombok.Getter;
+import org.springframework.stereotype.Component;
+
+import java.net.Socket;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Stores client data: socket connection, link for users and service port.
+ * Example: http://localhost:2222 -> http://sub1.localhost:9000
+ */
+@Getter
+@Component
+public class ClientList {
```

Comment:
Every method should have O(1) complexity

Reply:
Why? And how can I do it if i must iterate collection?

Reply:
Map<String, Socket>